lib/vmdb/plugins.rb

```diff
@@ -54,10 +56,22 @@ def register_provider_plugin(engine)
       end
     end
 
+    def registered_content_directories(engine, subfolder)
+      Dir.glob(engine.root.join("content", subfolder, "*")).each do |content_directory|
+        yield content_directory
+      end
+    end
+
     def register_automate_domains(engine)
-      Dir.glob(engine.root.join("content", "automate", "*")).each do |domain_directory|
+      registered_content_directories(engine, "automate") do |domain_directory|
         @registered_automate_domains << AutomateDomain.new(domain_directory)
       end
     end
+
+    def register_embedded_ansible_content(engine)
+      registered_content_directories(engine, "ansible") do |domain_directory|
```

Comment:
The `domain_directory` is a carry-over from the automate method.  Suggest `content_directory` for this one.